docs/developers/rate_limiting.md

```diff
@@ -0,0 +1,48 @@
+---
+id: rate_limiting
+title: Rate Limiting
+sidebar_label: Rate Limiting
+---
+
```

Comment:
There should be at least one line at the top of this page describing what's contained in the following table. Maybe something like:

```suggestion

These are the rate limits applied to the creation of all kinds of resources stored within the LMS's database. These limits are meant to prevents users from abusing resource creation end-points, and are meant to be unrealistic numbers to reach under normal circumstances.

```